Formula/consul.rb

```diff
@@ -78,6 +81,11 @@ def install
     end
 
     cd "src/github.com/hashicorp/consul" do
+      inreplace "scripts/build.sh" do |s|
+        s.gsub! "386 amd64 arm arm64", "amd64"
```

Comment:
Can't we pass that through the `XC_ARCH` and `XC_OS` env variables rather than hacking the build scripts?